tests/unit/s2n_cttime_functions_test.c

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
Have you seen the tests in s2n_safety_test.c? is there anything covered in the new tests not covered there?